azext_hanaonazure/custom.py

```diff
@@ -15,11 +15,16 @@ def list_hanainstance(client, resource_group_name=None):
         return client.list()
     return client.list_by_resource_group(resource_group_name)
 
+
 def restart_hanainstance(client, resource_group_name, instance_name):
     # The restart hanainstance REST API is a POST with no body.
     # The HANA RP API requires the Content-Type to be set.
     # Swagger does not add the Content-Type in the header if there is no body.
     # We need to add a custom header to force the API call to add the Content-Type.
     custom_header = {}
     custom_header['Content-Type'] = 'application/json; charset=utf-8'
-    return client.restart(resource_group_name, instance_name, custom_header)
\ No newline at end of file
+    return client.restart(resource_group_name, instance_name, custom_header)
+
+
+def update_hanainstance(client, resource_group_name, instance_name, **kwargs):
```

Comment:
@Doug, I know we discussed changing **kwargs to hana_instance as the input argument here. I tried that, but when I execute "az hanainstance update ..." commands, it will complain about missing "hana_instance" parameter in the command.

So I decided to extract the **kwargs for tags in the body of the method instead.

Reply:
(+1)